cms/static/js/views/pages/paged_container.js

```diff
@@ -28,11 +28,13 @@ define(['jquery', 'underscore', 'gettext', 'js/views/pages/container', 'js/views
 
             refreshXBlock: function(element, block_added, is_duplicate) {
                 var xblockElement = this.findXBlockElement(element),
-                    rootLocator = this.xblockView.model.id;
+                    rootLocator = this.xblockView.model.id,
+                    renderParameters;
                 if (xblockElement.length === 0 || xblockElement.data('locator') === rootLocator) {
                     this.render({refresh: true, block_added: block_added});
                 } else {
-                    this.refreshChildXBlock(xblockElement, block_added, is_duplicate);
+                    renderParameters = {can_edit_visibility: false};
```

Comment:
@cahrens Agreed that it would be best to contain disabling of visibility editing in library-specific code. But unless I'm missing something, I'm already doing that here:

There are two files called `paged_container.js`, `cms/static/js/views/pages/paged_container.js` and `cms/static/js/views/paged_container.js`. The code in this file (`cms/static/js/views/pages/paged_container.js`) seems to be library-specific: On library pages, it is [pulled in](https://github.com/open-craft/edx-platform/blob/itsjeyd/fix-library-visibility/cms/static/js/factories/library.js#L2) by `cms/static/js/factories/library.js`. It is not referenced anywhere else. `library.js`, in turn, is [pulled in](https://github.com/open-craft/edx-platform/blob/itsjeyd/fix-library-visibility/cms/templates/library.html#L29) by the `cms/templates/library.html` template for library pages, and is also not referenced by any other JS modules that implement client-side functionality for Studio (as you would expect). That should mean that the code in `pages/paged_container.js` is only used on library pages.

(It was introduced as part of the main content libraries implementation, #6459.)